src/index.js

```diff
@@ -81,4 +81,40 @@ function RebillyExperimentalAPI({apiKey = null, sandbox = false, timeout = baseT
     return createExperimentalApiInstance({apiHandler});
 }
 
-export {Errors as RebillyErrors, RebillyExperimentalAPI, cancellation};
+/**
+ * Create an instance of the experimental Rebilly API
+ * @param apiKey {string} private API key; if provided will be used for all requests
```

Comment:
I'm confused at how is a private API key used with a storefront?

Reply:
This is improper and I'll update